packages/wdio-applitools-service/src/index.js

```diff
@@ -0,0 +1,73 @@
+import logger from 'wdio-logger'
+import { Eyes, Target } from '@applitools/eyes.webdriverio'
+
+const log = logger('wdio-applitools-service')
+
+const DEFAULT_VIEWPORT = {
+    width: 1440,
+    height: 900
+}
+
+export default class ApplitoolsService {
+    constructor () {
+        this.eyes = new Eyes()
+    }
+
+    /**
+     * set API key in onPrepare hook and start test
+     */
+    beforeSession (config) {
+        const key = config.applitoolsKey || process.env.APPLITOOLS_KEY
```

Comment:
Should these two be switched so that it either takes the environment variable and if none is passed it will check for the property at the config?